agents/mysql/slowlog/slowlog.go

```diff
@@ -325,10 +328,17 @@ func makeBuckets(agentID string, res event.Result, periodStart time.Time, period
 			},
 			Mysql: &agentpb.MetricsBucket_MySQL{},
 		}
+
 		if v.Example != nil {
 			mb.Common.Example = v.Example.Query
 			mb.Common.ExampleFormat = agentpb.ExampleFormat_EXAMPLE
 			mb.Common.ExampleType = agentpb.ExampleType_RANDOM
+
+			// Skip query examples.
+			if disableQueryExamples {
+				mb.Common.Example = ""
```

Comment:
looks strange, maybe it's better to use if !disableQueryExample and set example values inside?